### ISSUE TITLE
Unexpected aggregation temporality for process.runtime.jvm.gc.duration

### ISSUE THREAD

**Issue by PeterF778** [open-telemetry/opentelemetry-java-instrumentation#7273]
Using java agent 1.20 and looking at the new histogram-based metric `process.runtime.jvm.gc.duration`.

It is reported as _cumulative_ metric, which seems to be much less useful than it would be with aggregation temporality of _delta_. In particular, the _min_ and _max_ values are almost useless, as there's no way to figure out how long ago they had been encountered. There's also no way they can be correctly recovered by any metric post-processor when attempting to convert the metric to _deltas_.

**Comment by trask** [comment]
hey @PeterF778, can you use `OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE=delta`? (https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk_exporters/otlp.md#additional-configuration)

**Comment by trask** [comment]
I added this to Thu SIG agenda to discuss

**Comment by fstab** [comment]
I'm wondering if you could use Exemplars. Exemplars are a way to attach example observations to metrics. So if you have an interesting observation (like the longest GC pause over the past 5 minutes) you might want to attach it as an Exemplar to the histogram.

Exemplars are often used for linking to example traces. However, the concept is very generic. The data model is just an observation (GC pause time) and an optional list of key/value pairs for additional info.

I'm not sure if the SDK supports custom Exemplars, but with the OTLP data model for metrics this would be perfectly possible, and it would also be compatible with Prometheus.

**Comment by PeterF778** [comment]
@fstab An interesting idea, but we do not have data for the longest GC pause over a recent interval. A histogram calculates this value automatically, there's no need for any additional code on the data collection side.

**Comment by trask** [comment]
Related to https://github.com/open-telemetry/semantic-conventions/issues/274

**Comment by trask** [comment]
@PeterF778 do you think your concern could be resolved by https://github.com/open-telemetry/semantic-conventions/issues/274? (we will be discussing potential bucket boundaries in tomorrow's SIG meeting)

**Comment by PeterF778** [comment]
> @PeterF778 do you think your concern could be resolved by [open-telemetry/semantic-conventions#274](https://github.com/open-telemetry/semantic-conventions/issues/274)? (we will be discussing potential bucket boundaries in tomorrow's SIG meeting)

Not really, I think these are separate issues.

**Comment by trask** [comment]
I don't think we can do anything about the default OTLP exporter temporality since that's part of stable spec. @PeterF778 is it ok to close this? thx

**Comment by PeterF778** [comment]
Yes, if we cannot do anything about it, it makes no sense to keep it open.
